Makefile

```diff
@@ -1,22 +1,67 @@
+#! /usr/bin/make -f
+
+# Project variables.
+PROJECT_NAME = starport
 DATE := $(shell date '+%Y-%m-%dT%H:%M:%S')
 VERSION = development
 HEAD = $(shell git rev-parse HEAD)
 LD_FLAGS = -X github.com/tendermint/starport/starport/internal/version.Version='$(VERSION)' \
 	-X github.com/tendermint/starport/starport/internal/version.Head='$(HEAD)' \
 	-X github.com/tendermint/starport/starport/internal/version.Date='$(DATE)'
 BUILD_FLAGS = -mod=readonly -ldflags='$(LD_FLAGS)'
+BUILD_FOLDER = ./dist
+BUILD_CMD = ./starport/cmd/starport
```

Comment:
```suggestion
```

go build ./... will automatically find the main pkg.

Reply:
if we use the `go build ./...` we cannot specify an output (`-o`)

![Screen Shot 2021-06-29 at 12 27 09](https://user-images.githubusercontent.com/2406457/123825499-558efd00-d8d5-11eb-9a67-8ce21253fe69.png)

Reply:
Can you try running `mkdir -p dist/starport` first?

Reply:
yes, sure! and the `-o` is already set, using this flag we cannot use the notation `./...`

Reply:
@go build $(BUILD_FLAGS) -o $(BUILD_FOLDER) ./...

Reply:
the `gen-cli-docs` binary will be built and put into the dist folder. For me it's ok, I don't see to much usage for this binary, but we can keep it